rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -132,7 +132,7 @@ object SystemProcesses {
         case isContractCall(
             produce,
             Seq(RhoType.String("fromPublicKey"), RhoType.ByteArray(publicKey), ack)
-        ) =>
+            ) =>
```

Comment:
lololololol

Reply:
Todays's discussion! That's why I'm so frustrated with the whole formatting discussion :)